tdd_activity/spec/solver_spec.rb

```diff
@@ -0,0 +1,33 @@
+require './tdd_activity/solver'
+
+describe Solver do
+  subject { Solver.new }
+
+  it 'calculate factorial of a number' do
+    expect(subject.factorial(4)).to eq(24)
+  end
+
```

Comment:
Great job team🤝

- [ ] As stated in the project requirements, for the Factorial method, kindly add the test for the special case - `when input is 0, the result should equal 1`.